1-exercises/D-do-while/exercise.js

```diff
@@ -8,6 +8,14 @@
 
 function evenNumbersSum(n) {
     // TODO
+    let i = 0;
+    let message = "Total";
+    let capture = "";
+    do {
+      let sum = message + capture[i];
+      console.log(sum);
+      i * 2;
+    } while (i < n);
```

Comment:
Not quite right. Please review the stopping condition and also the type of the variable `capture`